.gitignore

```diff
@@ -1,13 +1,16 @@
-# built assets
-/build
```

Comment:
@braican can you explain a little about the build and gitignore situation? I see the the webpack-generated `/build` folder is included in the Git repo, but I don't know why. Is that necessary for the continuous integration system?

Reply:
As far as I've been able to tell, this is a necessary evil type of thing. When using composer to pull this package, it grabs the code directly from this repo as is - there's no way to run a build step on install or anything like that (also keep in mind that Composer is meant for PHP packages and we're talking about built production assets via JS, so we're kind of crossing streams a bit here). Because of this, I've been unable to identify a good way to exclude the built assets from source control while also making them available in the downloadable package via composer - which means that the built assets need to be included in this repo.

I'm happy to find another way to ship build assets, because I really hate needing to include them in the repo as it could lead to merge issues and confusion.

Somewhat related, I mentioned this in the PR description but it's worth calling out again:

> To prevent the need for developers to remember to always run the npm run build command before each commit, I've set up a GitHub Action that is triggered by PRs that will run the build command and then commit the built assets back to the PR branch. In a vacuum this seems like it'll work but I'm curious about potential issues around merge conflicts moving forward...might be something we'll have to wait and see about.

Reply:
Ah, I didn't read the PR description!